packages/cli/src/util/alias/find-alias-by-alias-or-id.ts

```diff
@@ -1,17 +1,17 @@
 import { Output } from '../output';
 import { Alias } from '../../types';
+
 import Client from '../client';
 
 export default async function findAliasByAliasOrId(
   output: Output,
   client: Client,
   aliasOrId: string
 ) {
-  return client.fetch<Alias | null>(
+  return client.fetch<Alias>(
```

Comment:
```suggestion
  return client.fetch<Alias | null>(
```

This can return null if the alias was not found

Reply:
Oh nevermind, if we get a 200 it should never be null 👍

Reply:
Yeah I checked and the early version of this API returns 204 status with null body when not found

Reply:
Thanks for checking this :)